sys-devel/clang-crossdev-wrappers/clang-crossdev-wrappers-18.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 2022-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit crossdev
+
+DESCRIPTION="Symlinks to a Clang crosscompiler"
+HOMEPAGE="https://wiki.gentoo.org/wiki/Project:LLVM"
+SRC_URI=""
+S=${WORKDIR}
+
+LICENSE="public-domain"
+SLOT="${PV}"
+KEYWORDS=""
+PROPERTIES="live"
+
+RDEPEND="
+	sys-devel/clang:${SLOT}
+"
+
+src_compile() {
```

Comment:
Let's do all of this in src_install with heredocs with `newbin`:
`newbin - << ....`